Lib/test/test_compile.py

```diff
@@ -1181,6 +1181,27 @@ def test_complex_single_line_expression(self):
         self.assertOpcodeSourcePositionIs(compiled_code, 'BINARY_OP',
             line=1, end_line=1, column=0, end_column=27, occurrence=4)
 
+    def test_multiline_assert_rewritten_as_method_call(self):
+        # GH-94694: Copying location information from a "real" node to a
+        # handwritten one should always be valid!
```

Comment:
Should it really always be valid?

Reply:
I can remove the comment if it's too controversial. It just seems to me that the utility/safety of `ast.copy_location` and `ast.fix_missing_locations` is dramatically reduced if they can't at least promise crash-free behavior.